tests/integration/test_orgs_invites.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""Integration tests for invitation methods implemented on orgs."""
+import github3
+
+from .helper import IntegrationHelper
+
+
+class TestInvitations(IntegrationHelper):
+
+    """Team integration tests."""
+
+    betamax_kwargs = {'match_requests_on': ['method', 'uri', 'json-body']}
+
+    def setUp(self):
+        super(TestInvitations, self).setUp()
+        self.basic_login()
+
+    def get_organization(self, name='mozillatw'):
+        """Get the organization for each test."""
+        self.token_login()
+        o = self.gh.organization(name)
+        assert isinstance(o, github3.orgs.Organization)
+        return o
+
+    def test_invites(self):
```

Comment:
Okay - here I need some guidance. I'm new to working with cassettes and betamax. One reason I separated them was to make it easy to generate the cassette for these new cases.

As a noob - it looked to me like a number of the TestOrganization cassettes relied upon a specifically crafted test org. I.e. I would not be able to get a successful run. Similarly for these tests - for it to be meaningful, at least one invitation needs to be pending.

I imagine this is a common occurrence for new features added to an existing API endpoint. Is there a way to merge cassettes? Ideally, I'd like a workflow where:
 - I can easily show I haven't broken the existing implementation (including tests)
 - I can iterate as needed on generating cassette coverage for my new work

For regression tests, you ask for the tests and cassettes to not be merged in. With a bit of hand waving, a new feature a new feature could be viewed as a regression of the old implementation. (Okay, a lot of hand waving and some rose colored glasses.) Would it be reasonable to use that approach (with the test & class renaming that implies) for small pieces like this?